modules/database/sqlite/wp-includes/sqlite/class-perflab-sqlite-pdo-engine.php

```diff
@@ -326,7 +326,7 @@ private function prepare_directory() {
 		global $wpdb;
 		$u = umask( 0000 );
 		if ( ! is_dir( FQDBDIR ) ) {
-			if ( ! @mkdir( FQDBDIR, 0704, true ) ) {
+			if ( ! @mkdir( FQDBDIR, 0704, true ) ) { // phpcs:disable WordPress.PHP.NoSilencedErrors.Discouraged
```

Comment:
If we're ignoring WordPress-Extra checks for the SQLite module in our `phpcs.xml.dist`, then let's not modify any files in that module.

Reply:
👍